source/reference/command/replSetReconfig.txt

```diff
@@ -53,15 +53,28 @@ considerations:
 - A majority of the set's members must be operational for the
   changes to propagate properly.
 
-- This command can cause downtime as the set renegotiates
-  primary-status. Typically this is 10-20 seconds, but could
-  be as long as a minute or more. Therefore, you should attempt
-  to reconfigure only during scheduled maintenance periods.
-
-- In some cases, :dbcommand:`replSetReconfig` forces the current
-  primary to step down, initiating an election for primary among the
-  members of the replica set. When this happens, the primary node will drop
-  all current connections.
+- The :method:`rs.reconfig()` shell method can trigger the current 
```

Comment:
This bullet is in the replSetReconfig page. This makes it sound like if you run the command replSetReconfig, no primary stepdown, whereas if you use the shell method, well, hello stepdown.